api/proxy/metrics/metrics.go

```diff
@@ -110,15 +106,15 @@ func NewMetrics(subsystem string) *Metrics {
 		}, []string{
 			"method", // no status on histograms because those are very expensive
 		}),
-		SecondaryRequestsTotal: factory.NewCounterVec(prometheus.CounterOpts{
+		SecondaryRequestsTotal: promauto.With(registry).NewCounterVec(prometheus.CounterOpts{
```

Comment:
@samlaf @litt3 would it be beneficial to update our [wait-for](http://github.com/Layr-Labs/eigenda/blob/master/.github/workflows/test-proxy.yml#L180) to also check for existence of metrics being exposed on the `/metrics` endpoint?